server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/MIMEMessageConverter.java

```diff
@@ -67,6 +67,7 @@
 import com.google.common.io.ByteStreams;
 import com.google.common.net.MediaType;
 
+@SuppressWarnings("deprecation")
```

Comment:
Why do we use something deprecated. are you sure we can't avoid it instead of removing the warning ?